critiquebrainz/db/review.py

```diff
@@ -40,6 +40,21 @@ def to_dict(review, confidential=False):
     return review
 
 
+# Returns:
+#     List of the top ten languages (only the language codes) in CritiqueBrainz.
+def get_top_languages():
+    with db.engine.connect() as connection:
+        result = connection.execute(sqlalchemy.text("""
+               SELECT  review.language
+                 FROM review
+             GROUP BY review.language
+             ORDER BY COUNT(review.id) DESC
+                LIMIT 10
+            """))
+        languages = result.fetchall()
+    return languages
```

Comment:
This now returns a list of just the `RowProxy` elements, which leads to the error on building the development environment:
```
critiquebrainz_1        | Traceback (most recent call last):
critiquebrainz_1        |   File "manage.py", line 14, in <module>
critiquebrainz_1        |     application = DispatcherMiddleware(frontend.create_app(), {
critiquebrainz_1        |   File "/code/critiquebrainz/frontend/__init__.py", line 109, in create_app
critiquebrainz_1        |     from critiquebrainz.frontend.views.review import review_bp
critiquebrainz_1        |   File "/code/critiquebrainz/frontend/views/review.py", line 14, in <module>
critiquebrainz_1        |     from critiquebrainz.frontend.forms.review import ReviewCreateForm, ReviewEditForm, ReviewReportForm
critiquebrainz_1        |   File "/code/critiquebrainz/frontend/forms/review.py", line 42, in <module>
critiquebrainz_1        |     FREQUENTLY_USED_LANGUAGES.append((language_code, Locale(language_code).language_name))
critiquebrainz_1        |   File "/usr/local/lib/python3.6/site-packages/babel/core.py", line 166, in __init__
critiquebrainz_1        |     identifier = str(self)
critiquebrainz_1        |   File "/usr/local/lib/python3.6/site-packages/babel/core.py", line 359, in __str__
critiquebrainz_1        |     self.script, self.variant))
critiquebrainz_1        |   File "/usr/local/lib/python3.6/site-packages/babel/core.py", line 1133, in get_locale_identifier
critiquebrainz_1        |     return sep.join(filter(None, (lang, script, territory, variant)))
critiquebrainz_1        | TypeError: sequence item 0: expected str instance, RowProxy found
docker_critiquebrainz_1 exited with code 1
```